addons/l10n_it_edi/data/invoice_it_simplified_template.xml

```diff
@@ -18,80 +18,87 @@
         <template id="account_invoice_it_simplified_FatturaPA_export">
             <t t-set="currency" t-value="record.currency_id or record.company_currency_id"/>
             <t t-set="bank" t-value="record.partner_bank_id"/>
-            <p:FatturaElettronicaSemplificata  t-att-versione="formato_trasmissione" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:p="http://ivaservizi.agenziaentrate.gov.it/docs/xsd/fatture/v1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ivaservizi.agenziaentrate.gov.it/docs/xsd/fatture/v1.0 http://www.fatturapa.gov.it/export/fatturazione/sdi/fatturapa/v1.0/Schema_del_file_xml_FatturaPA_versione_1.0.xsd">
-            <FatturaElettronicaHeader>
+            <p:FatturaElettronicaSemplificata t-att-versione="formato_trasmissione" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:p="http://ivaservizi.agenziaentrate.gov.it/docs/xsd/fatture/v1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ivaservizi.agenziaentrate.gov.it/docs/xsd/fatture/v1.0 http://www.fatturapa.gov.it/export/fatturazione/sdi/fatturapa/v1.0/Schema_del_file_xml_FatturaPA_versione_1.0.xsd">
+                <FatturaElettronicaHeader>
                     <DatiTrasmissione>
                         <IdTrasmittente>
-                            <IdPaese t-out="get_vat_country(record.company_id.vat)"/>
-                            <IdCodice t-out="normalize_codice_fiscale(record.company_id.l10n_it_codice_fiscale) or get_vat_number(record.company_id.vat)"/>
+                            <IdPaese t-out="sender_partner.country_code"/>
+                            <IdCodice t-out="sender_partner.vat"/>
```

Comment:
Are not the NamedTuples a bit too magical?
```suggestion
                            <IdCodice t-out="sender_partner['vat']"/>
```

Reply:
I don't think they're magical, they're quite used throughout Odoo.
Yet now that I think again about it -- even though freezing keys may be good for us to determine what's in and what's not -- namedtuples are much harder to extend if at anytime we want to inherit methods and add info.